k8s/common/video-hearings.yaml

```diff
@@ -1,7 +0,0 @@
----
```

Comment:
this will cause the vh namespace to get deleted, probably needs:

1. moving (can be done later)
2. reverting for now

Reply:
Agreed. I'd thought to remove not to have these deployed in prod but yes, will impact lower envs.

Reply:
Branch now rolled back to previous approved commit.